package.json

```diff
@@ -24,10 +24,13 @@
     "webpack-dev-server": "^1.16.2"
   },
   "dependencies": {
+    "express": "^4.15.2",
```

Comment:
This should be a devDependency since express is not required to use the loader package.